client/app/assets/less/inc/misc.less

```diff
@@ -225,4 +225,13 @@
     height: 37px;
     border-radius: 2px;
     width: 37px;
+}
+
+/* --------------------------------------------------------
+    Percy
+-----------------------------------------------------------*/
+@media only percy {
+    .hide-in-percy, .pace {
+        display: none;

```

Comment:
`display: none;` seemed better, but tell me if you think we get some value on having a white space

Reply:
There's no difference to Percy comparison.
But visibility keeps the layout unaltered while display might mess things up, which might confuse a human looking at the images. So I favor `visibility: hidden`.

You can even go as far as this, to have full clarity 😬:
```css
@media only percy {
  .hide-in-percy, .pace {
    visibility: hidden;
    position: relative;

    &:after {
      content: 'Hidden from Percy';
      font-size: 10px;
      color: #000;
      opacity: 0.5;
      visibility: visible;
      position: absolute;
      left: 2px;
      top: 2px;
    }
  }
}
```

Reply:
> You can even go as far as this, to have full clarity grimacing (only if you want to)

Nah, this may create issues when we want to hide small things.